flexget/components/notify/notifiers/discord.py

```diff
@@ -115,15 +120,39 @@ def notify(self, title, message, config):
         :param dict config: discord plugin config
         """
 
-        web_hook = {
-            'content': message,
-            'username': config.get('username'),
-            'avatar_url': config.get('avatar_url'),
-            'embeds': config.get('embeds'),
-        }
+        for embed in config.get('embeds', []):
+            ts = embed.get('timestamp')
+            if ts:
+                if isinstance(ts, str):
+                    if ts.isdigit():
+                        try:
+                            ts = datetime.utcfromtimestamp(int(ts))
+                        except (ValueError, OverflowError):
+                            logger.info(
+                                f"Value provided for 'timestamp' ({embed['timestamp']}) "
+                                f"is not a timestamp ({int(datetime.now().timestamp())})."
+                            )
+                    else:
+                        try:
+                            ts = isoparse(ts)
+                            embed['timestamp'] = ts
+                        except (ParserError, ValueError) as e:
+                            logger.info(f"'timestamp' is in an invalid format: {e}")
+                if not isinstance(ts, datetime):
+                    embed.pop('timestamp', None)
+                    logger.warning("'timestamp' is invalid, dropping it")
+                else:
+                    embed['timestamp'] = datetime.strftime(ts, r'%Y-%m-%dT%H:%M:%S%z')
+
+            web_hook = {
```

Comment:
Is this bit indented too much? Shouldn't this just happen once after the loop?

Reply:
There can be up to 10 embeds in a single message, the loop parses them and evaluates each `timestamp` property

Reply:
Right, but the web hook is only used once after all the embeds are processed, and contains all of them.

Reply:
Ah, I see now. Indeed the web_hook definition was a little too indented. The code preview misled me since it was the last line